src/__tests__/publish.test.js

```diff
@@ -54,7 +54,11 @@ describe('PUBLISH command', () => {
 
   it('allows publishing from main', async () => {
     git._setBranch('main');
-    await publish(merge(NOMINAL_OPTIONS));
+    try {
+      await publish(merge(NOMINAL_OPTIONS));
+    } catch (err) {
+      throw new Error('SHOULD_NOT_HAVE_THROWN');
+    }
```

Comment:
What's the purpose of this try-catch block? If the `publish()` fails, the test will already fail…

Reply:
Ok I'll remove it